src/message_interface/mod.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+mod account_method;
+mod message;
+mod message_handler;
+mod message_type;
+mod response;
+mod response_type;
+
+pub use account_method::AccountMethod;
+pub use message::Message;
+pub use message_handler::WalletMessageHandler;
+pub use message_type::{AccountToCreate, MessageType};
+pub use response::Response;
+pub use response_type::ResponseType;
+
+use crate::{account_manager::AccountManager, Result};
+use tokio::sync::mpsc::unbounded_channel;
+
+pub async fn create_message_handler(path: Option<String>) -> Result<WalletMessageHandler> {
```

Comment:
I think we should also allow to set other manager options here, what do you think about adding Serialize to  `AccountManagerBuilder` and use that with a method `AccountManger::from_str()` or something similar? Would require some change to the SignerHandle, so we can also build it from a String

Reply:
Right! Setting manager options here makes total sense, I'll play with some of the ideas here but then the SignerHandle changes would have to come from iota.rs.

Reply:
Currently for node.js binding we have this, I guess we want something similar
https://github.com/iotaledger/wallet.rs/blob/53d0fe2c2ce8390620f6c1b1f311e46e6ba05a03/bindings/nodejs/src/types.rs#L157
Regarding iota.rs you're right, I can also do that part

Reply:
Thanks! I've made other corrections